test/girouette/tw/border_test.cljc

```diff
@@ -5,6 +5,113 @@
 (deftest component-test
   (are [class-name expected-garden]
     (= expected-garden (class-name->garden class-name))
+    ;; Border Radius
+    "rounded-none"
+    [".rounded-none" {:border-radius "0px"}]
 
-    "p-0"
-    [".p-0" {:padding "0"}]))
+    "rounded"
+    [".rounded" {:border-radius "0.25rem"}]
+
+    "rounded-xl"
+    [".rounded-xl" {:border-radius "0.75rem"}]
+
+    "rounded-t-sm"
+    [".rounded-t-sm" {:border-top-left-radius  "0.125rem"
+                      :border-top-right-radius "0.125rem"}]
+
+    "rounded-tl-full"
+    [".rounded-tl-full" {:border-top-left-radius "9999px"}]
+
+    ;; Border Width
+    "border"
+    [".border" {:border-width "1px"}]
+
+    "border-2"
+    [".border-2" {:border-width "2px"}]
+
+    "border-t"
+    [".border-t" {:border-top-width "1px"}]
+
+    "border-l-4"
+    [".border-l-4" {:border-left-width "4px"}]
+
+    "border-b-2em"
+    [".border-b-2em" {:border-bottom-width "2em"}]
+
+    "border-3px"
+    [".border-3px" {:border-width "3px"}]
+
+    ;; Border Color
+    "border-transparent"
+    [".border-transparent" {:border-color "transparent"}]
+
+    "border-green-300"
+    [".border-green-300" {:--gi-border-opacity 1
+                          :border-color        "rgba(134, 239, 172, var(--gi-border-opacity))"}]
+
+    ;; Border Opacity
+    "border-opacity-25"
+    [".border-opacity-25" {:--gi-border-opacity 0.25}]
+
+    ;; Border Style
+    "border-dotted"
+    [".border-dotted" {:border-style "dotted"}]
+
+    ;; Divide Width
+    "divide-x"
+    [#garden.selectors.CSSSelector{:selector ".divide-x > * + *"}
+     {;;:--gi-divide-x-reverse 0 ;; Not implemented
+      :border-right-width "calc(1px * var(--gi-divide-x-reverse))"
+      :border-left-width  "calc(1px * calc(1 - var(--gi-divide-x-reverse))"}]
+
+    "divide-y-2"
+    [#garden.selectors.CSSSelector{:selector ".divide-y-2 > * + *"}
+     {;;:--gi-divide-x-reverse 0 ;; Not implemented
+      :border-top-width "calc(2px * var(--gi-divide-y-reverse))"
+      :border-bottom-width  "calc(2px * calc(1 - var(--gi-divide-y-reverse))"}]
+
+    ;; Divide Color
+    "divide-current"
+    [#garden.selectors.CSSSelector{:selector ".divide-current > * + *"}
+     {:border-color "currentColor"}]
+
+    "divide-gray-100"
+    [#garden.selectors.CSSSelector{:selector ".divide-gray-100 > * + *"}
+     {:--gi-divide-opacity 1
+      :border-color        "rgba(244, 244, 245, var(--gi-divide-opacity))"}]
+
+    ;; Divide Opacity
+    "divide-opacity-70"
+    [".divide-opacity-70" {:--gi-divide-opacity 0.7}]
+
+    ;; Divide Style
+    "divide-double"
+    [#garden.selectors.CSSSelector{:selector ".divide-double > * + *"}
+     {:border-style "double"}]
+
+    ;; Ring Width
+    ;; TODO: Test for `*	box-shadow: 0 0 #0000;`
```

Comment:
I will take care of this case. Leave this comment here, it will be useful later.